services/tenant-ui/frontend/src/components/issuance/Schemas.vue

```diff
@@ -29,14 +30,28 @@
       <Column field="attributes" header="Attributes" />
       <Column field="schema_id" header="ID" />
     </DataTable>
-    <Button
-      class="create-btn"
-      icon="pi pi-plus"
-      label="Create Schema"
-      @click="toggleAddSchema"
-    />
+    <div class="row buttons">
+      <Button
+        class="create-btn"
+        icon="pi pi-plus"
+        label="Create Schema"
+        @click="toggleAddSchema"
+      />
+      <Button
+        class="copy-btn"
+        icon="pi pi-copy"
+        label="Copy Schema"
+        @click="toggleCopySchema"
+      />
+    </div>
     <Dialog header="Create a new schema" v-model:visible="displayAddingSchema">
-      <CreateSchema @schema-save="toggleAddSchema"></CreateSchema>
+      <CreateSchema @schema-save="schemaCreated"></CreateSchema>
+    </Dialog>
+    <Dialog
+      header="Copy an existing schema"
+      v-model:visible="displayCopyingSchema"
```

Comment:
Can shorthand without the `v-model:` for atrributes with just `:visible`

Reply:
Love it!